soufi/finders/yum.py

```diff
@@ -230,7 +231,20 @@ def do_task(target, *args):
     """Run the target callable in a subprocess and return its response."""
     queue = Queue()
     process = Process(target=target, args=(queue,) + args)
-    process.start()
+    try:
+        process.start()
+    except RuntimeError:
+        sys.exit(
+            """
+FATAL: Running this finder directly from the global scope is not supported on
+       this platform.  To use this finder, call it instead from the main
+       module, e.g.:
+
+           if __name__ == '__main__':
+               soufi.finder.factory(*args, **kwargs).find()
+
+Aborting."""
+        )
```

Comment:
Also please consider using `dedent` so we can indent the error string :)

Reply:
The behavior will surface on any platform that uses `spawn` instead of `fork` in its `multiprocessing` module, which today means Windows and Mac, and could mean other platforms in the future.

But `multiprocessing` is prone to throw `RuntimeError`s for lots of reasons, so a good-faith check that it's whinging about this particular problem is not unreasonable.

And I suppose calling `textwrap.dedent` is similarly not too much to ask for 😆